### DISCUSSION TITLE
Tardis dependencies

### DISCUSSION THREAD

**Discussion by DilanRona** [eccentricdevotion/TARDIS#469] [Q&A] [answered]
I need to update the tardis plugin, and last time I had issues with the worldedit and worldguard having to be a specific version for the tardis plugin to work.

What are the versions for worldedit and worldguard that i have to download in order to guarantee that the tardis plugin will work correctly?

**Comment by Chew** [comment, marked as answer]
Every version should be the latest, no reason to not be.

Specifically:

- WorldGuard 7.0.0 or above. The latest is 7.0.4 I believe.
- WorldEdit is a dependency for WorldGuard, not TARDIS. The latest is 7.2.3 I believe.

**Comment by DilanRona** [comment]
ok, thanks.
